src/routes/posts/index.json.ts

```diff
@@ -1,21 +1,17 @@
 import db from '$lib/database';
 
 interface Body {
-	body: {
-		posts: Array<Post>;
-	};
+  body: {
+    posts: Array<Post>;
+  };
 }
 
 export async function get(): Promise<Body> {
-	const posts: Array<Post> = db.prepare('select * from posts').all();
+  const posts: Array<Post> = db.prepare('select * from posts').all();
```

Comment:
This may be a problem once I write a lot of posts. Better add `LIMIT 10` for now and we'll deal with pagination later on.